LICENSE

```diff
@@ -1,11 +1,4 @@
-YEAR: 2016
-COPYRIGHT HOLDER: Liye He
-
-and specify as
-
-License: MIT + file LICENSE
-
-Copyright (c) <2016>, <Liye He>
+Copyright (c) <2016>, <Liye He>, <Google, Inc.>
```

Comment:
Please note: Google claims copyright and ownership rights over my contribution to this project.

Reply:
To include Google as copyright holder now is under discussion in the legal department of university of helsinki. I will get back to you when I know more. I am so sorry for the late reply due to my trip to Oslo and my supervisor's trip to Spain.